app/views/hub/clients/edit_13614c_form_page2.html.erb

```diff
@@ -229,7 +229,7 @@ scope of service chart)", yes_no_options_for_select) %>
         <!-- COMMENTS -->
         <hr style="margin: 5px;"/>
         <div class="grid" style="background-color:#e5e5e5; padding: 10px; display: block; align-items: flex-start; justify-content: space-between;">
-            <%= f.cfa_input_field(:cv_p2_notes_comments, "Notes/Comments", classes: ["form-width--long"]) %>
+            <%= f.cfa_textarea(:cv_p2_notes_comments, "Notes/Comments", options: { rows: 2, columns: 100 }) %>
```

Comment:
I think I'm actually covering this in a story I have right now: https://github.com/codeforamerica/vita-min/pull/5620/files